test/parallel/test-buffer-mask.js

```diff
@@ -0,0 +1,92 @@
+var common = require('../common');
+var assert = require('assert');
+
+var tests = [
+  testBasic,
+  testBounds,
+  testOverflow,
+  testNonAligned,
+  testReturnValue
+]
+
+tests.forEach(Function.prototype.call.bind(Function.prototype.call))
```

Comment:
Clever... but a hell of a lot longer than `tests.forEach(function(f) { f(); })` :-)

Minor style nit: missing semicolon.